packages/spotlight/src/container.js

```diff
@@ -740,7 +740,8 @@ function getContainerFocusTarget (containerId) {
 			return getContainerFocusTarget(nextId);
 		}
 
-		return element;
+		const {overflow} = getContainerConfig(containerId);
+		return !overflow ? element : contains(getContainerRect(containerId), getRect(element)) && element;
```

Comment:
This ternary is a bit hard to read with the additional `&&` at the end. WDYT of breaking it up to:

```
if (!overflow || contains(getContainerRect(containerId), getRect(element))) {
  return element;
}
```